consensus/enclave/impl/src/lib.rs

```diff
@@ -131,6 +133,9 @@ pub struct SgxConsensusEnclave {
     /// is valid. To ensure that it is uniform across the network, it's hash
     /// gets appended to responder id.
     blockchain_config: OnceBox<BlockchainConfigWithDigest>,
+
+    /// Constant time minimum fee map, initialized from blockchain config
+    ct_min_fee_map: OnceBox<CtTokenMap<u64>>,
```

Comment:
We do see to be adding a lot of new vars to core structs. At some point we should have some accounting from the perspective of both code organization and  to do some broader benching/testing to sanity check we haven't introduced any side effects. 

Probably near release of stablecoin and token minting we review this.

Reply:
yah so, we have a pattern now with `BlockchainConfig` being config parameters to the enclave, and then we had `BlockchainConfigWithDigest` being like, that, plus some cached stuff. This `CtTokenMap` could also be part of the cached stuff and we could reorganize mroe, i just didn't want to start taking that apart right now

Reply:
Agreed, if we decide to do that, it should be definitely later in a refactoring task. Is it worth making an issue for?

Reply:
sounds good to me!